lib/IDE/ExprContextAnalysis.cpp

```diff
@@ -712,6 +713,8 @@ class ExprContextAnalyzer {
 } // end anonymous namespace
 
 ExprContextInfo::ExprContextInfo(DeclContext *DC, Expr *TargetExpr) {
+  if (isa<Initializer>(DC))
```

Comment:
Why not handle this deeper in the ExprContextAnalyzer, or in the parent finding?

Reply:
No reason. I moved it into the parent finding, because that seems to be more correct.